code/game/objects/items/storage/boxes.dm

```diff
@@ -594,11 +594,11 @@
 	STR.can_hold = typecacheof(list(/obj/item/reagent_containers/food/snacks/monkeycube))
 
 /obj/item/storage/box/mixedcubes/PopulateContents()
-	for(var/i in 1 to 3)
```

Comment:
Sir? This isn't randomization, this is a for loop

Reply:
i think im retarded